src/Sarif.WorkItemFiling/Sarif.WorkItemFiling.csproj

```diff
@@ -4,9 +4,16 @@
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard2.0|AnyCPU'">
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+    <NoWarn>NU5105kk</NoWarn>
```

Comment:
>kk [](start = 18, length = 2)

I'm curious what this rule means, never seen one with kk at the end before.

Reply:
That's a mistake, this code shouldn't have been added. the kk is a typo. Thanks for catching this, not sure how I missed it on my preliminary review...

---
In reply to: [376502450](https://github.com/microsoft/sarif-sdk/pull/1746#discussion_r376502450) [](ancestors = 376502450)